docs: Clarify Logits Processor and TRT-LLM examples (#116)
triton-inference-server/tutorials: Popular_Models_Guide/Hermes-2-Pro-Llama-3-8B/README.md

3.  Launch Tritonserver

> [!NOTE]

> > This tutorial was prepared for serving TensorRT-LLM model on a single GPU.
> > Thus, in the following command use `--world_size=1`, if you also built
> > an engine for a single GPU. Alternatively, if the engine requires multi-GPU
> > scenario, make sure to specify the exact number of GPU, required by you engine
> > in `--world_size`.

Reviewer: wording tweak
```suggestion
> This tutorial was prepared for serving a TensorRT-LLM model on a single GPU.
> Thus, in the following command use `--world_size=1` if the engine was built
> for a single GPU. Alternatively, if the engine requires multiple GPUs
> make sure to specify the exact number of GPUs required by the engine
> in `--world_size`.
```
